docs/en/OLED_Display.md

```diff
@@ -0,0 +1,212 @@
+    powersave_off_time=30, # time in seconds to turn off screen
+)
+```
+
+# Image
+
+Before starting make sure you got a ready file to display. It should be a monochromatic bitmap (for example `1.bmp`) with same size as your's OLED. In this case 128x64. 
+Place your image straight into your CircuitPython file, next to `main.py`. 
+#### Placing it in separate folder may cause issues.
+
+```python
+oled.entries = [
+    ImageEntry(image="1.bmp", x=0, y=0),
+]
+keyboard.extensions.append(oled)
+```
+
+You can also make your images appear corresponding to specific layer
+
+```python
+oled.entries = [
+    ImageEntry(image="1.bmp", x=0, y=0, layer=0),
+    ImageEntry(image="2.bmp", x=0, y=0, layer=1),
+]
+keyboard.extensions.append(oled)
+```
+
+And/or side of your split keyboard
+
+```python
+oled.entries = [
+    ImageEntry(image="L1.bmp", x=0, y=0, side="L"),
+    ImageEntry(image="R1.bmp", x=0, y=0, side="R"),
+]
+keyboard.extensions.append(oled)
+```
+
+# Text
+To display longer texts you need to divide it to smaller phrases to fit width of your screen.
+
+You're able to freely positon of your text to place it wherever you want just by changing x and y values
+
+```python
+oled.entries = [
+    TextEntry(text="Layer = 1", x=0, y=0),
+    TextEntry(text="Macros", x=0, y=12),
+    TextEntry(text="Hey there!", x=0, y=24),
+]
+keyboard.extensions.append(oled)
+```
+### X and Y anchors
+It's helpfull especially with positioning of text.
+The values can be set "T" for Top, "M" for Middle and "B" for Bottom for X axis as well as "L" for Left, "M" for Middle and "R" for Right for Y axis.
+for more precise displaying, the values can be set in range from 0 to 1.
+It sets the anchor point of given text and your text is moved and placed basing on this anchor point.
+For example for text in top right corner you need to set its anchor points Top Right and move text to far right position.
+For some more info about anchors check [Adafruit site](https://learn.adafruit.com/circuitpython-display-support-using-displayio/text).
+
+```python
+oled.entries = [
+    TextEntry(text="Layer = 1", x=128, y=0, x_anchor="R", y_anchor="T"), # text in Top Right corner
+    TextEntry(text="Macros", x=128, y=64, x_anchor="R", y_anchor="B"), # text in Bottom Right corner
+    TextEntry(text="Hey there!", x=64, y=32, x_anchor="M", y_anchor="M"), # text in the Middle of screen
+]
+keyboard.extensions.append(oled)
+```
+
+### Split
+As well as with images you can change displaying according to your layer or side of split keyboard.
+
+```python
+oled.entries = [
+    TextEntry(text="Longer text that", x=0, y=0, layer=0),
+    TextEntry(text="has been divided", x=0, y=12, layer=0, side="L"),
+    TextEntry(text="for an example", x=0, y=24, layer=0, side="R"),
+]
+keyboard.extensions.append(oled)
+```
+
+### Line Spacing
+Doesn't serve any purpose beside looks, might as well be skipped - default spacing of 0.75 will be applied
```

Comment:
Is only relevant when line break character is used, the usage of line break character must be demonstrated.

Or spacing can be skipped entirely since it's really not that useful.